parameters.cpp

```diff
@@ -261,7 +261,7 @@ bool Parameters::addParameters(){
    // NOTE Do not remove the : before the list of variable names as this is parsed by tools/check_vlasiator_cfg.sh
    Readparameters::addComposing("variables.diagnostic", std::string()+"List of data reduction operators (DROs) to add to the diagnostic runtime output. Each variable to be added has to be on a new line diagnostic = XXX. Names are case insensitive. "+
 				"Available (20190320): "+
-				"populations_blocks "+
+				"populations_vg_blocks "+
```

Comment:
You may note that diagnostic variables don't have vg_ or fg_ as they are not saved on either grid, actually.

Reply:
But it is confusing when running the cfg check tool. Nad it does relate to vg cells and not fg cells.